test/images/clusterapi-tester/Makefile

```diff
@@ -12,20 +12,15 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-all: push
+BIN=clusterapi-tester
+ARCH ?= amd64
+TARGET ?= $(CURDIR)
+GOLANG_VERSION ?= latest
 
-# 0.0 shouldn't clobber any released builds
-TAG = 1.0
-PREFIX = gcr.io/google_containers/clusterapi-tester
-
-main: main.go
-	CGO_ENABLED=0 GOOS=linux go build -a -installsuffix cgo -ldflags '-w' -o main ./main.go
-
-container: main
-	docker build --pull -t $(PREFIX):$(TAG) .
-
-push: container
-	gcloud docker -- push $(PREFIX):$(TAG)
-
-clean:
-	rm -f main
+bin:
```

Comment:
we probably should also have `.PHONY: bin` in each of these, though it probably won't matter in practice.